src/components/editCqlLibrary/EditCqlLibrary.tsx

```diff
@@ -812,9 +817,7 @@ const EditCqlLibrary = () => {
             onClose={handleDialogClose}
             onSubmit={createVersionLibrary}
             cqlLibraryError={null}
-            isCqlPresent={
-              loadedCqlLibrary && loadedCqlLibrary.cql?.trim().length > 0
-            }
+            isCqlPresent={loadedCqlLibrary && loadedCqlLibrary.cql?.length > 0}
```

Comment:
I think it is better to put the trim because there might situation where user can keep pressing enter and save it

Reply:
I agree. it was throwing errors in my test, but I figured out why and put it back